content/legacy/3_representation.md

```diff
@@ -0,0 +1,8 @@
+---
+image: legacy/representation.png
+headline: Representation of precipitation extremes
+---
+
+## Representation of precipitation extremes
+
+Regional Climate Models integrated on convection permitting grids (dx<=4km; cpRCMs) improve the representation of precipitation extremes, as shown by Pichelli et al. (2021), e.g. in their Figure 3. However, information about the properties of the events, causing these extremes, is not yielded.
```

Comment:
Can we make the references link to the corresponding doi's?

Reply:
Done!